packages/backend/seeds/schedules-and-shifts.ts

```diff
@@ -39,50 +39,49 @@ export async function seed(knex: Knex): Promise<void> {
   ]);
```

Comment:
So what happens when we change the seed file? Does it have some diffing algorithm to regenerate the database of shifts? Or do we basically need to drop / recreate the DB when that happens?

Reply:
I dont think its very intelligent. In my perspective the seed data is primarily for dev.